### PR TITLE
Nerfs Ash Drake loot

### PR DESCRIPTION
**Pull request by ghost**
apparently you can stack the fucking resistances which i don't want since complete immortality

anyway that's gone, as with everything else. no more staff of 'unremovable instant kill', no more wands of fireball, nothing

all of that shit's OP so it's gone

if your argument is 'but then nobody will fight the drake' I really don't give a fuck because that's not my concern, you fight the drake because you want the challenge, not so that you can get fantastic gear to kill antags with

you want that, go to /tg/

##### Changelog

:cl:
rscadd: The ash drake blood potion has been nerfed. Again. Also, all of the other ash drake drops are removed.
/:cl:

### CODE REVIEW

**Comment by Jc1469** [comment]
I'm partially fine with gutting the drops, my own changes to megafauna could add drops in for harder versions regardless.  But.

[4:02 PM] Johncena1469: >now only drops blood
[4:03 PM] Johncena1469: >blood is now more useless than paper
[4:03 PM] Johncena1469: [Deletion of Content]

File: code/modules/mining/lavaland/necropolis_chests.dm [hunk 1]
```diff
@@ -455,17 +453,7 @@
...
-		if(1)
-			new /obj/item/weapon/melee/ghost_sword(src)
-		if(2)
-			new /obj/item/weapon/lava_staff(src)
-		if(3)
-			new /obj/item/weapon/spellbook/oneuse/fireball(src)
-			new /obj/item/weapon/gun/magic/wand/fireball(src)
-		if(4)
-			new /obj/item/weapon/dragons_blood(src)
+		new /obj/item/weapon/dragons_blood(src)
```
Comment (Jc1469) [inline code/modules/mining/lavaland/necropolis_chests.dm]: Remove a tab or it wont compile. re.

File: code/modules/mining/lavaland/necropolis_chests.dm [hunk 2]
```diff
@@ -562,18 +550,8 @@
...
-		if(1)
-			user << "<span class='danger'>You feel robust!</span>"
-			var/datum/species/S = user.dna.species
-			S.brutemod *= 0.5
-			S.burnmod *= 0.5
-			S.coldmod *= 0.5
-		if(2)
-			user << "<span class='danger'>You feel like you could walk straight through lava now.</span>"
-			H.weather_immunities |= "lava"
+		user << "<span class='danger'>You feel like you could walk straight through lava now.</span>"
```
Comment (Jc1469) [inline code/modules/mining/lavaland/necropolis_chests.dm]: re

File: code/modules/mining/lavaland/necropolis_chests.dm [hunk 3]
```diff
@@ -562,18 +550,8 @@
...
+		H.weather_immunities |= "lava"
```
Comment (Jc1469) [inline code/modules/mining/lavaland/necropolis_chests.dm]: also re

File: code/modules/mining/lavaland/necropolis_chests.dm (hunk 2 above)
Comment (Jc1469) [inline code/modules/mining/lavaland/necropolis_chests.dm]: indent

File: code/modules/mining/lavaland/necropolis_chests.dm (hunk 3 above)
Comment (Jc1469) [inline code/modules/mining/lavaland/necropolis_chests.dm]: indent pls.

File: code/modules/mining/lavaland/necropolis_chests.dm [hunk 4]
```diff
@@ -633,14 +632,14 @@
...
+	desc = "A magically infused bottle of blood, imbued with powerful healing properties."
 	icon = 'icons/obj/wizard.dmi'
 	icon_state = "vial"
 
 /obj/item/mayhem/attack_self(mob/user)
 	for(var/mob/living/carbon/human/H in range(7,user))
 		spawn()
-			var/obj/effect/mine/pickup/bloodbath/B = new(H)
+			var/obj/effect/mine/pickup/healing = new(H)
 			B.mineEffect(H)
```
Comment (Jc1469) [inline code/modules/mining/lavaland/necropolis_chests.dm]: what is B

**Comment by GrayRachnid** [comment]
Why are you removing guardian spirits from lavaland?

**Comment by Jc1469** [comment]
Would it kill you to just try to compile it for once and see where the issues actually are, instead of people having to point out every single one, or someone like Dark having to come in and fix it for you?

**Comment by ghost** [comment]
Because guardian spirits are holoparasites, and holoparasites are bad.

For the same reason they were removed from traitors nonantags shouldn't have them.

File: code/modules/mining/lavaland/necropolis_chests.dm [hunk 5]
```diff
@@ -655,14 +632,14 @@
 
 /obj/item/mayhem
 	name = "mayhem in a bottle"
```
Comment (Jc1469) [inline code/modules/mining/lavaland/necropolis_chests.dm]: mmh.  I doubt this is getting merged regardless, but even so, new item type maybe.  new name maybe.  healing so mayhem much wow

**Comment by GrayRachnid** [comment]
That's not really a reason, holoparasites were removed from traitors because they caused too much chaos. Guardian spirits however are held by non-antags, therefore not much chaos can be caused. If guardian spirits are issues then so is the die of fate, so is bubblegum's chainsaw, and so is the lavastaff, the ash storm staff. Guardian spirits should not be removed, they are a lavaland items, they are not related to the station or the syndicate.

**Comment by ghost** [comment]
The issue with holoparasites is that they are unstunnable abominations that do boatloads of damage and cannot be anticipated.

They cause balance issues in that non-antag station crew with holoparasites are massively stronger than most antagonists by sheer virtue of having a holoparasite that does loads of damage and depending on the different abilities can wipe out a small crowd working alongside a competent partner.

It causes balance issues, especially when a miner returns to station with their overpowered gear and begins antagonist hunting. If anything the guardiancreator drop is stronger than most megafauna drops like the spectral sword from the ash drake (which does not drop anymore) because of it's sheer power and utility.

The lavastaff is also being removed. Bubblegum's chainsaw is simply a melee weapon and counterable as such. The die of fate is a one use item that has barely any utility and is rarely obtainable at all.

Guardian spirits WILL be removed, because regardless of being 'a lavaland items' they are extremely powerful tools far beyond anything either most antagonists or station crew can get their hands on.

They were removed from traitor not just because they caused too much chaos but also because they A.

Didn't fit the theme of traitor

and B.

Were retardedly powerful.

The same issues apply to nonantags/an antag getting them on lavaland. Except with nonantags getting them it's the same problem except the crew fights alongside them, amplifying their power, and with antagonists such as cultists getting them most of the issues with holoparasites can be averted by the other relevant antagonist abilities.

Altogether, they're being removed unless you can provide me with a well written and comprehensive reason not to.

**Comment by GrayRachnid** [comment]
If you start removing lavaland items then i'll start adding overpowered lavaland items. The current balance state of lavaland is completely fine, and the chances of finding a guardian spirit is one every 100 rounds. There is absolutely no point in removing the guardian spirit as it poses no threat to the balance of the station, and let's be honest. Most of the item and guardian spirit will die with it's miner on lavaland, never to see the station to begin with.

**Comment by ghost** [comment]
That statement is more full of shit than a toilet on taco bell night.

Lavaland's balance state is not fine and anyone who's played it for half a second knows this.

You can't add anything to it either, so that's an empty threat.

Make all the pointless incorrect statements you want, the fact remains that lavaland shit is op and dumb. First go the megafauna drops.

Then most of the ruin loot on the advice of stealthkibbler who they themselves added most of the ruins and admitted they're op and dumb.

And then whatever else I feel like.

And there's shit all you have say in the matter.

**Comment by GrayRachnid** [comment]
And john is striving to make megafauna harder, why don't you collaborate with him? 

My point is, nothing is wrong with guardian spirits. I can't be bothered writing an essay for you to give you my opinion. 

Guardian spirits are fine, they're rare, they're not that OP, and anybody who gets them will be happy and nobody will be sad.

**Comment by ghost** [comment]
Except for the antags who get steamrolled by them.

I don't see a reason to keep them in there.

**Comment by GrayRachnid** [comment]
You have a higher chance to get an ash storm staff than a guardian spirit, yet the staff can fuck over more people.

**Comment by Rafael450** [comment]
As i always say, I bet that in the next nerf we will change the name from YOGSTATION to BARBIESTATION, and i think it will look amazing, MLP lobby songs look cool too

**Comment by ShadowDeath6** [comment]
keep holos.

meathook is more op than holos and the majority of station items in general and it's not removed

you also generally "can't anticipate" being esworded in the dick by a miner with a pirate esword but hey you can get that in chests too and it's way more dangerous since you can get stunned by it and die in 4 hits

File: code/modules/mining/lavaland/necropolis_chests.dm [hunk 6]
```diff
@@ -672,15 +649,7 @@
...
-	..()
-	var/loot = rand(1,3)
-	switch(loot)
-		if(1)
-			new /obj/item/weapon/antag_spawner/slaughter_demon(src)
-		if(2)
-			new /obj/item/bloodvial/bloodcrawl(src)
-		if(3)
-			new /obj/item/bloodvial/saw(src)
+	new /obj/item/bloodvial/saw(src)
```
Comment (ShadowDeath6) [inline code/modules/mining/lavaland/necropolis_chests.dm]: nice undocumented changes lel

**Comment by AdamElTablawy** [comment]
i can remove meathook and the pirate sword too

**Review by ShadowDeath6 — DISMISSED** [review]
provide actual reasons rather than outright removing with no documentation or explanation of features other coders put work into adding

File: code/modules/mining/lavaland/necropolis_chests.dm (hunk 6 above)
Comment (Jc1469) [reply code/modules/mining/lavaland/necropolis_chests.dm]: if you're going to requestchange meme, request changes for the one I spoke about above.

Comment (ShadowDeath6) [reply code/modules/mining/lavaland/necropolis_chests.dm]: that was actually a mistake on my part

**Comment by ShadowDeath6** [comment]
furthermore I'm getting extremely tired of you saying "oh lol I'm removing [thing] because I said so and if you disagree you can suck a chode". It's extremely unprofessional and frankly the attitude you've shown to people giving genuine advice is fucking abhorrent and I'm surprised you've been given this many strikes considering you never fix your shit when someone says "stop using the online editor and getting help from people until your code is functional"

**Comment by ghost** [comment]
I'm not a professional at any point Shadow.

I code for an online space station game on my free time. I'm not an actual coder.

People giving genuine advice is one thing. People who don't have any concept of what they're working with is another. In the rare, rare circumstance that someone actually provides feedback I'm receptive to it. 

But they never do. It's always just shitposts.

**Comment by ghost** [comment]
jesus christ did it comment 10 times let me delete

**Comment by ShadowDeath6** [comment]
>the chances of finding a guardian spirit is one every 100 rounds. There is absolutely no point in removing the guardian spirit as it poses no threat to the balance of the station, and let's be honest. Most of the item and guardian spirit will die with it's miner on lavaland, never to see the station to begin with.

an actual coherent comment, wow.

you respond with:
>That statement is more full of shit than a toilet on taco bell night.
First go the megafauna drops.
Then most of the ruin loot on the advice of stealthkibbler who they themselves added most of the ruins and admitted they're op and dumb.
And then whatever else I feel like.
And there's shit all you have say in the matter.

People don't need to argue for a reason not to merge this pr, YOU should argue for a reason for our headcoders to merge the pr. You have the same shit power that anybody else has as long as you continue behaving like a brat who yells at people until they agree with you.

**Comment by ghost** [comment]
There are no undocumented changes and everything has been adressed above.

For bubblegum:
The demon vial is essentially a free grief tool that you can smash for essentially no benefit and gets you banned for using it and unleashing a demon.

The bloodcrawl vial, loathe as I am to remove it because I really like bloodcrawling, lends itself to some balance concerns since it, unlike every other item in the game, means that the user can become completely immortal at will and sink into the (usually plentiful) blood everywhere. They cannot be touched or killed by anything and can see anywhere. For obvious reasons this is more imbalanced than any other item available in mining. If you have an idea on how to weaken it i'd love to hear it-even a delay before entering or leaving would be great.

The mayhem in a bottle/contract are also essentially grief tools that don't really have a reason to exist, as i'm sure you are aware. One makes everyone kill each other for no reason, the other makes everything kill one guy for no reason. Things like that feel like wizard spells or admin commands, not random-findable loot.

**Comment by ghost** [comment]
As for gray's comment-

None of the points I raised earlier were adressed at all. His only argument to keep the item was, to sum it up, 'it is rare so it's fine.'

Which is, not really fine.

Numerous necropolis tendrils spawn every round. Most of them are hunted down. Of those, there's a fairly high chance that within the next 5 rounds someone's gonna get a guardian spirit.

It's not as though it's a guaranteed drop. I've never said that. The times that it doesn't affect the balance of the station are very large.

But when it does, it has an impact. Any antagonist seeking to assassinate the miner with the guardian or a group antagonist in general now has their work cut out for them due to having what is essentially a living crew-sided antagonist weapon, on someone who is already probably armed to the teeth.

It's simply nonsensical to have it. Gray raised a few points but none of them adressed my concern. Hence, my response.

**Comment by ShadowDeath6** [comment]
The mayhem potion and the contract were both removed for obvious reasons when I added the new bubblegum loot. The point is that you get one vial, and you don't know whether or not you're going to get something good or bad, hence the point of "chaos". Slaughter demons are random events that can occur whether or not people do anything otherwise. It's a stupid idea to remove the entire point of the feature so you can get the most dull item every single time.

and BEFORE you say "oh well it spices up the round because it's dangerous and should be removed because of that", there are a ton more dangerous things than a non antag miner with a holoparasite or bloodcrawl. If they validhunt with their loot then it's in admin territory to deal with it. 

Unless it's something completely round disrupting and extremely common like ashwalkers breaking into the station and trying to kill everyone every other round it's not a code concern.

File: code/modules/mining/lavaland/necropolis_chests.dm [hunk 7]
```diff
@@ -655,14 +632,14 @@
 
 /obj/item/mayhem
 	name = "mayhem in a bottle"
-	desc = "A magically infused bottle of blood, the scent of which will drive anyone nearby into a murderous frenzy."
+	desc = "A magically infused bottle of blood, imbued with powerful healing properties."
```
Comment (Jc1469) [inline code/modules/mining/lavaland/necropolis_chests.dm]: If this is completely replacing the bottle of **mayhem**, make a new item entirely.  Healing isn't exactly mayhem.

**Review by Jc1469 — CHANGES_REQUESTED** [review]

**Comment by ghost** [comment]
Fine, you win. YOu want your demons and holoparas, you can have them, I guess. Nothing I do can convince you how retardedly stupid it is.

File: code/modules/mining/lavaland/necropolis_chests.dm [hunk 8]
```diff
@@ -66,6 +64,9 @@
 			new /obj/item/weapon/spellbook/oneuse/summonitem(src)
 		if(23)
 			new /obj/item/organ/heart/cursed/wizard(src)
+		if(24)  
+			new /obj/item/weapon/guardiancreator(src)
+
```
Comment (ShadowDeath6) [inline code/modules/mining/lavaland/necropolis_chests.dm]: might want to put this back at 14 to avoid problems

**Review by Jc1469 — APPROVED** [review]
you couldve just added a subtype to the potion, or just made another random crappy item that has the same effect, instead of just reverting it.

**Comment by ghost** [comment]
honestly i have issue with the items existing on lavaland but i can't remove the items while the bosses still exist

so the solution is to remove the bosses

**Comment by Super3222** [comment]
honestly you could have just set those values instead of multiplying them

**Comment by ShadowDeath6** [comment]
Slime person with resist to every damage is pretty memey though

File: code/modules/mining/lavaland/necropolis_chests.dm (hunk 8 above)
Comment (Cruix) [reply code/modules/mining/lavaland/necropolis_chests.dm]: yes

Comment (ShadowDeath6) [reply code/modules/mining/lavaland/necropolis_chests.dm]: Still gotta fix this

**Comment by ShadowDeath6** [comment]
Changed title to actually tell what the PR did, as it did more than change the blood

**Comment by Jc1469** [comment]
@Cruix If you're going to keep cleaning up his code for him, atleast change the 1, 24 to 1,  23.  There's only 23 things in the pool after all.

**Comment by Cruix** [comment]
He has already reduced the chance of getting nothing by one, that's enough undocumented change.

**Pull request merged**